pkg/rancher-desktop/window/index.ts

```diff
@@ -56,6 +57,15 @@ export function getWindow(name: string): Electron.BrowserWindow | null {
   return (name in windowMapping) ? BrowserWindow.fromId(windowMapping[name]) : null;
 }
 
+/**
+ * Close all open windows.
+ */
+function closeAll() {
+  for (const name in windowMapping) {
```

Comment:
It might be better to use [`BrowserWindow.getAllWindows()`](https://www.electronjs.org/docs/latest/api/browser-window#browserwindowgetallwindows) in case there are unregistered windows (e.g. child windows of another window, dialogs, etc.).

Alternatively, use [`Electron.app.quit()`](https://www.electronjs.org/docs/latest/api/app#appquit) instead.  We had handling for `window-all-closed` to hide the dock icon if we're going to keep running without any windows open, but that does not apply in this case.